projects/kit/components/pdf-viewer/pdf-viewer.service.ts

```diff
@@ -0,0 +1,21 @@
+import {Injectable} from '@angular/core';
+import {AbstractTuiDialogService} from '@taiga-ui/cdk';
+import {PolymorpheusComponent} from '@tinkoff/ng-polymorpheus';
+import {TuiPdfViewerOptions} from './pdf-viewer-options';
+import {TuiPdfViewerComponent} from './pdf-viewer.component';
+
+const DIALOG = new PolymorpheusComponent(TuiPdfViewerComponent);
+const DEFAULT_OPTIONS = {
```

Comment:
what do you think?

```ts
const DEFAULT_OPTIONS = { label: '', actions: '' } as const;
```

Reply:
I thought prettier takes care of it

Reply:
Anyway, let's not comment on such things. If prettier thinks it's ok — we think it's ok